components/PasswordDialog.qml

```diff
@@ -136,12 +137,33 @@ Item {
                     color: "black"
 
                     Image {
-                        width: 12
-                        height: 16
-                        source: "../images/lockIcon.png"
+                        width: 26
+                        height: 26
+                        opacity: 0.7
+                        fillMode: Image.PreserveAspectFit
+                        source: isHidden ? "../images/eyeShow.png" : "../images/eyeHide.png"
                         anchors.verticalCenter: parent.verticalCenter
                         anchors.right: parent.right
                         anchors.rightMargin: 20
+                        MouseArea {
+                            anchors.fill: parent
+                            cursorShape: Qt.PointingHandCursor
+                            hoverEnabled: true
+                            onClicked: {
+                                passwordInput.echoMode = isHidden ? TextInput.Normal : TextInput.Password;
+                                isHidden = !isHidden;
+                            }
+                            onEntered: {
+                                parent.opacity = 0.9
+                                parent.width = 28
```

Comment:
All integer constants should terminate with `* scaleRatio`.

Reply:
Updated. Thank you.